sw/device/tests/meson.build

```diff
@@ -917,6 +917,13 @@ subdir('autogen')
 ###############################################################################
 subdir('sim_dv')
 
+###############################################################################
+# Closed Source Test Libraries
+###############################################################################
+if get_option('build_closed_source')
+  subdir('closed_source')
+endif
```

Comment:
This should probably be moved to `$REPO_TOP/meson.build`? Not sure if it is a good idea to attach the closed source tree to `sw/device/tests` parent. Better to subdir from REPO root so that the closed source entry point could be anywhere in the repo.

Reply:
It is better, but the meson file under closed_source depends on the meson file inside tests dir. I'm not a Meson expert, if it can be done please share an example.

Reply:
Again, a problem with meson, I don't think it's worth shuffling a bunch of meson `subdir` orderings around to hack this together, when it will be resolved cleanly with bazel. Marking as resolved.